docs/manage/remotes.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Parts, Sub-Parts and Remotes"
+linkTitle: "Remotes and Sub-Parts"
+weight: 40
+type: "docs"
+description: "Connect robots to each other."
+tags: ["server", "components", "services"]
+---
+
+When robots can communicate with each other, they can share resources and operate collaboratively.
+This document explains how to establish secure connections between robots.
+
+### Robot parts
+
+Robots are organized into *parts*, where each part represents a computer (a single-board computer like a Raspberry Pi or a desktop, laptop, or other computer) running `viam-server`, the hardware [components](/components/) attached to it, and any [services](/services/) or other resources running on it.
+Every robot has a main part.
+Multi-part robots also have one or more *sub-parts* representing additional computers running `viam-server`.
+
+There are two ways to link robot parts:
+
+- **Sub-part**: If you have two computers within the *same robot*, use one as the main part and [connect the other to it as a sub-part](#configure-a-sub-part).
+
+- **Remote**: To connect two computers that are parts of *different robots*, [add one robot part as a remote part of the other robot](#configure-a-remote).
+
+Connections between robots are established using the best network path available.
+
+When you configure a remote or a sub-part, the main robot part can access all the components and services configured on the remote robot part as though they were entities of the main robot part.
+This is a one-way connection: The main robot part can access the resources of the remote robot part, but the remote robot cannot access the resources of the robot part remoting into it.
+
+<img src="../img/remotes/remotes-diagram.png" alt="Example of a remote and a two part robot where the main (and only) part of robot 1 remotes into the main part of robot 2, and thus has access to all resources of robot 2." >

```

Comment:
Okay I don't like saying that a part configures something since I think of the human as doing the configuring. I will change it to "Part A of Robot 1 has Part A of Robot 2 as a remote part." 

I don't understand what X would be. Resource?